packages/helpers/src/index.ts

```diff
@@ -375,6 +376,51 @@ export function createTransactionFromSkeleton(
   return tx;
 }
 
+type Promisable<T> = Promise<T> | T;
+export type LiveCellFetcher = (outPoint: OutPoint) => Promisable<Cell>;
+
+export async function createTransactionSkeleton(
```

Comment:
```suggestion
/**
* create a {@link TransactionSkeleton} from a {@link Transaction}
* @example 
* ```js
* const fetcher = new RPC('localhost:8114').getLiveCell
* const skeleton = await createTransactionSkeleton({ transaction, fetcher });
* ```
*/
export async function createTransactionSkeleton(
```